### ISSUE TITLE
FirstWithoutOrderByAndFilterWarning logged when querying byte[] property

### ISSUE THREAD

**Issue by AndriySvyryd** [aspnet/EntityFrameworkCore#9372]
```C#
context.ProductPhotos.Select(c => c.Photo.First().ToString()).ToList()
```

See MonsterFixupFixtureBase

**Comment by smitpatel** [comment]
But that is first without order by! What is the exact warning?

**Comment by AndriySvyryd** [comment]
`Photo` is a byte[], element order is always deterministic

**Comment by smitpatel** [comment]
Query pipeline is dumb :trollface:

**Comment by divega** [comment]
Check for types that don't need this check, e.g. ordered things like `byte[]` and `string`.

**Comment by maumar** [comment]
fixed in 0071bcc032ca42b6fa699d3eca2ab36b9da0e653
